terrain_test.py

```
@@ -0,0 +1,86 @@
+            # Regenerate terrain
+            terrain = Terrain(width, height)
+            stdscr.clear()
+            stdscr.addstr(0, 0, "Terrain Generation Test")
+            stdscr.addstr(1, 0, "Press 'r' to regenerate, 'q' to quit")
+
+
+if __name__ == "__main__":
+    try:
+        curses.wrapper(main)
+    except KeyboardInterrupt:
+        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(KeyboardInterrupt)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(KeyboardInterrupt)`

(SIM105)